apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/CommitRepository.java

```diff
@@ -35,7 +35,7 @@ List<Commit> findByAppIdAndClusterNameAndNamespaceNameAndDataChangeLastModifiedT
       String appId, String clusterName, String namespaceName, Date dataChangeLastModifiedTime, Pageable pageable);
 
   @Modifying
-  @Query("update Commit set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where appId=?1 and clusterName=?2 and namespaceName = ?3 and isDeleted = false")
+  @Query("update Commit set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where appId=?1 and clusterName=?2 and namespaceName = ?3 and IsDeleted = false")
```

Comment:
Is it necessary to change appId, clusterName and namespaceName to AppId, ClusterName and NamespaceName?
```suggestion
  @Query("update Commit set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where AppId=?1 and ClusterName=?2 and NamespaceName = ?3 and IsDeleted = false")
```

Reply:
@nobodyiam Thanks, fixed